scanners/boostsecurityio/baseline/module.yaml

```diff
@@ -0,0 +1,13 @@
+api_version: 1.0
+
+
+group: boostsecurityio/groups/baseline
+id: boostsecurityio/baseline
+name: BoostSecurity Scanner
+namespace: boostsecurityio/baseline
```

Comment:
I think you might be missing scan types, which probably be
```
- sast
- cicd
- metadata
- sci
```
(based on what is included)

Reply:
nope, those come from the includes

Reply:
Ok, that will require an update to the schema in the action, because that's why it's filing.

I can update the shared-models utils script to parse the inclusion

Reply:
sure that, but also the missing steps, and config. 
i can give it a shot tomorrow morning, probably easier since i'm familiar with the format